src/main/resources/log4j2.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration>
+	<ThresholdFilter level="all" />
+	<Appenders>
+		<Console name="STDOUT" target="SYSTEM_OUT">
+			<PatternLayout
+				pattern="[%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n" />
+		</Console>
+		<RollingFile name="RollingFile" fileName="${sys:logPath}/CC-LOG.log"
+			filePattern="${sys:logPath}/CC-LOG-%d{yyyy-MM-dd}-%i.log"
+			ImmediateFlush="true" AppendToFile="true">
```

Comment:
This is the description from the documentation:
>When set to true - the default, each write will be followed by a flush. This will guarantee the data is written to disk but could impact performance.
>Flushing after every write is only useful when using this appender with synchronous loggers. Asynchronous loggers and appenders will automatically flush at the end of a batch of events, even if immediateFlush is set to false. This also guarantees the data is written to disk but is more efficient.
I wanna say that our two appenders are synchronous, so it might be something worth leaving as-is. If we notice a performance hit from the client, we can always disable this.